README.md

```diff
@@ -1,21 +1,42 @@
 # Introduction to Building Front-End Applications with React
 
 ## Objectives
-
 1. Explain the topics this course will cover
 2. Describe this course's approach to React
 3. Briefly describe React
 
-## Overview
+## What you'll learn in this course
+This course is an introduction to React.js, one of the most popular front-end libraries at the moment, with its own very
+active and extensive ecosystem. Many front-end engineers use React.js to create amazing applications, and you'll be a
+part of that group yourself in no time!
 
-Walk students through this course. What topics will they cover? What should they
-already know?
+This course handles all the basic React stuff. We'll learn about JSX, props, state and how to structure our components.
+When we're done with all the lessons, you'll be able to create real applications with React, including fetching remote
+data and handling events.
 
-Also explain that this first unit _should_ be review for them — if they
-struggle, they can brush up before moving on. Put another way, let students
-know that it's okay for this first unit to be super easy — just want to make
-sure that they have the basics down before diving into anything new.
+## What you should already know
+This course assumes a working familiarity with JavaScript and HTML. The first section in this course will contain brief
+lessons on several core JavaScript techniques and paradigms that are widely used throughout React: the prevalence of
+functional programming, writing pure functions, fetching remote data, ... It's totally okay if you manage to fly through
+these lessons — they're supposed to be a refresher and not super challenging yet. You'll get to the good stuff in no
+time!
 
-## Resources
+## A tiny primer on React
```

Comment:
This section is great! We were just asked many of the questions that it answers in person a couple of days ago :)